packages/mdc-textfield/_mixins.scss

```diff
@@ -14,6 +14,31 @@
 // limitations under the License.
 //
 
+@mixin mdc-text-field-box-corner-radius($radius) {
+  border-radius: $radius $radius 0 0;
+}
+
+@mixin mdc-text-field-textarea-corner-radius($radius) {
+  border-radius: $radius;
+
+  .mdc-textfield__label {
+    border-radius: $radius;
```

Comment:
Might want to flatten the bottom of this to prevent parts of text from "poking" out with larger border radius values